### ISSUE TITLE
"Elixir.Inspect.DateTime' specified in timex and elixir" for timex ~> 3.1

### ISSUE THREAD

**Issue by GildedHonour** [bitwalker/timex#355]
I've run "deps.get --all" and it printed "timex 3.1.24" on a server. In mix.ex I have      {:timex, "~> 3.1"}

Yet this error keeps occuring:

```
==> Including ERTS 9.0 from /usr/lib/erlang/erts-9.0
==> Generating boot script
==> Release failed, during .boot generation:
        Duplicated modules:                                                  
        'Elixir.Inspect.DateTime' specified in timex and elixir    
```

**Comment by bitwalker** [comment]
What version of Elixir and what is the version of Timex in your mix.lock?

**Comment by GildedHonour** [comment]
elixir 1.5
timex  "3.1.21"

**Comment by bitwalker** [comment]
You should upgrade to 3.1.23, it fixes this incompatibility with Elixir 1.5
